doc/sdk_with_carto.md

```diff
@@ -55,10 +55,602 @@ This high-level workflow describes how to prepare your mobile data for rendering
+
+  CartoSQLService service = new CartoSQLService();
+  service.Username = "nutiteq";
+
+  PointStyleBuilder builder = new PointStyleBuilder
+  {
+    Color = new Carto.Graphics.Color(255, 0, 0, 255),
+    Size = 1
+  };
+
+  MapView.QueryFeatures(service, source, builder.BuildStyle(), query);
+
+  // Extension Method in a static class (shared code)
+  public static void QueryFeatures(this MapView map, CartoSQLService service, LocalVectorDataSource source, PointStyle style, string query)
+  {
+    System.Threading.Tasks.Task.Run(delegate
+    {
+      FeatureCollection features = service.QueryFeatures(query, map.Options.BaseProjection);
+
+      for (int i = 0; i < features.FeatureCount; i++)
+      {
+        Feature feature = features.GetFeature(i);
+
+        // This data set features point geometry,
+        // however, it can also be LineGeometry or PolygonGeometry
+        PointGeometry geometry = (PointGeometry)feature.Geometry;
+
+        var point = new Point(geometry, style);
+        source.Add(point);
+      }
+
+    }); 
+  }
+
+  {% endhighlight %}
+  </div>
+
+  <div class="Carousel-item js-Tabpanes-item--lang js-Tabpanes-item--lang--objective-c">
+  {% highlight objc %}
+
+  // Only get cities with over 100k, else it'll be too many
+  NSString *sql = @"SELECT * FROM cities15000 WHERE population > 100000";
+  
+  // Initialize CartoSQL service, set a username
+  NTCartoSQLService *service = [[NTCartoSQLService alloc] init];
+  [service setUsername:@"nutiteq"];
+  
+  NTPointStyleBuilder *builder = [[NTPointStyleBuilder alloc]init];
+  NTColor *color = [[NTColor alloc] initWithR:255 g:0 b:0 a:255];
+  [builder setColor:color];
+  [builder setSize:1];
+
+  NTPointStyle *style = [builder buildStyle];
+  
+  dispatch_queue_t queue = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0ul);
+  
+  // Set on background thread for "animated" point appear
+  dispatch_async(queue, ^{
+      NTFeatureCollection *features = [service queryFeatures:sql proj:self.projection];
+  
+      for (int i = 0; i < [features getFeatureCount]; i++) {
+
+          // This data set features point geometry,
+          // however, it can also be LineGeometry or PolygonGeometry
+          NTPointGeometry *geometry = (NTPointGeometry *)[[features getFeature:i] getGeometry];
+      
+          NTPoint *point = [[NTPoint alloc] initWithGeometry:geometry style:style];
+          [self.source add:point];
+      }
+  });
+
+  {% endhighlight %}
+  </div>
+</div>
+
+### CARTO.js API
+
+Reference: https://carto.com/docs/carto-engine/carto-js/
 
 In order to integrate your published mobile map into your app, you need a callback-based asynchronous process to request the viz.json.
 
+Warning! viz.json API is in development and experimental. Use at your own risk.
+
 1)  The following requirements are needed for app integration of a published CARTO Mobile SDK map:
 
   - Online API is requested and will run in another thread
```

Comment:
What is the "online API"? 
And how is this section different from https://carto.com/docs/carto-engine/mobile-sdk/sdk-with-other-carto-tools/#sdk-and-carto-apis (that starts on line 5-9 of this file)?